README.md

```diff
@@ -174,7 +174,7 @@ Hardware accelleration users for Nvidia will need to install the container runti
 
 https://github.com/NVIDIA/nvidia-docker
 
-We automatically add the necessary environment variables that will use all available GPU's on the host. Once nvidia-docker is installed on your host you will need to just start the docker with the nvidia container runtime ```--runtime=nvidia```. NVIDIA automatically mounts the GPU and drivers from your host into the plex docker.
+We automatically add the necessary environment variables that will use all available GPU's on the host. Once nvidia-docker is installed on your host you will need to just start the docker with the nvidia container runtime ```--runtime=nvidia``` and add an environment variable ```-e NVIDIA_VISIBLE_DEVICES=all``` (can also be set to a specific gpu's UUID). NVIDIA automatically mounts the GPU and drivers from your host into the plex docker.
```

Comment:
Is this statement still true?  Or do they need to add `-e NVIDIA_VISIBLE_DEVICES=all`  

We automatically add the necessary environment variable that will utilise all the features available on a GPUs on the host. Once nvidia-docker is installed on your host you will need to just start the docker with the nvidia container runtime ```--runtime=nvidia``` and add an environment variable ```-e NVIDIA_VISIBLE_DEVICES=all``` (can also be set to a specific gpu's UUID, this can be discovered by running `nvidia-smi --query-gpu=gpu_name,gpu_uuid --format=csv` ). NVIDIA automatically mounts the GPU and drivers from your host into the plex docker.

Reply:
updated